.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'error',
     'no-console': 'error',
+    'no-unused-expressions': ['error', { allowShortCircuit: true }],
```

Comment:
Let's not disable this. I agree with [this criticism](https://github.com/airbnb/javascript/issues/963#issuecomment-235048756) of readability.